includes/Modules/Analytics/Advanced_Tracking.php

```diff
@@ -53,6 +63,13 @@ final class Advanced_Tracking {
 	 */
 	private $plugin_detector;
 
+	/**
+	 * Main class code injector instance.
+	 *
+	 * @var Measurement_Code_Injector
+	 */
+	private $measurement_code_injector;
```

Comment:
This class property is unnecessary, since the instance is only used in one place.

Reply:
This issue has been resolved.